megamek/src/megamek/common/MechSummaryCache.java

```diff
@@ -245,9 +274,23 @@ public void loadMechData(boolean ignoreUnofficial) {
             bNeedsUpdate |= loadMechsFromDirectory(vMechs, sKnownFiles, lLastCheck, userDataUnits, ignoreUnofficial);
         }
 
+        // save updated cache back to disk
+        if (bNeedsUpdate) {
+            try {
+                saveCache();
+            } catch (Exception e) {
+                loadReport.append("  Unable to save mech cache\n");
+                MegaMek.getLogger().error(e);
+            }
+        }
```

Comment:
Possible location issue

Reply:
I don't know what you mean by that.

Reply:
As part of the previous comment on how I'd read the method name.